windows/win_get_url.py

```diff
@@ -96,18 +96,18 @@
 - name: Download earthrise.jpg to 'C:\\Users\\RandomUser\\earthrise.jpg'
   win_get_url:
     url: 'http://www.example.com/earthrise.jpg'
-    dest: 'C:\\Users\\RandomUser\\earthrise.jpg'
+    dest: 'C:\Users\RandomUser\earthrise.jpg'
 
-- name: Download earthrise.jpg to 'C:\\Users\\RandomUser\\earthrise.jpg' only if modified
+- name: Download earthrise.jpg to 'C:\Users\RandomUser\earthrise.jpg' only if modified
   win_get_url:
     url: 'http://www.example.com/earthrise.jpg'
-    dest: 'C:\\Users\\RandomUser\\earthrise.jpg'
+    dest: 'C:\Users\RandomUser\earthrise.jpg'
     force: no
 
-- name: Download earthrise.jpg to 'C:\\Users\\RandomUser\\earthrise.jpg' through a proxy server.
+- name: Download earthrise.jpg to 'C:\Users\RandomUser\earthrise.jpg' through a proxy server.
   win_get_url:
     url: 'http://www.example.com/earthrise.jpg'
-    dest: 'C:\\Users\\RandomUser\\earthrise.jpg'
+    dest: 'C:\Users\RandomUser\earthrise.jpg'
     proxy_url: 'http://10.0.0.1:8080'
     proxy_username: 'username'
     proxy_password: 'password'
```

Comment:
I'd just nuke the quotes on all these- we'll handle that in the "how to do Windows paths" doc fragment.